src/components/Tabs/ScrollableTabs/ScrollableTabs.tsx

```diff
@@ -198,18 +188,36 @@ export class ScrollableTabs extends React.Component<
     this._animateScroll(scrollLeft + scrollDistance);
   }
 
-  getNavScrollPosition(): ScrollPosition {
-    const scrollPosition = { scrollLeft: 0, scrollRight: 0 };
+  getNavButtons(): NavButtonOptions {
+    const { rtl } = this.props;
+    let navButtons = NavButtonOptions.none;
 
     if (this._navRef && this._navRef.current) {
       const element = this._navRef.current;
-
-      scrollPosition.scrollLeft = element.scrollLeft;
-      scrollPosition.scrollRight =
-        element.scrollWidth - (element.scrollLeft + element.offsetWidth);
+      const { offsetWidth, scrollLeft, scrollWidth } = element;
+      if (rtl) {
+        if (scrollWidth !== offsetWidth + Math.abs(scrollLeft) + 1) {
+          navButtons =
+            scrollLeft === 0 ? NavButtonOptions.left : NavButtonOptions.both;
+        } else {
+          navButtons = NavButtonOptions.right;
+        }
+      } else {
+        if (scrollLeft >= SCROLL_EPSILON) {
+          navButtons = NavButtonOptions.left;
+        }
+        const scrollRight =
+          element.scrollWidth - (element.scrollLeft + element.offsetWidth);
+        if (scrollRight >= SCROLL_EPSILON) {
+          navButtons =
+            navButtons === NavButtonOptions.none
+              ? NavButtonOptions.right
+              : NavButtonOptions.both;
+        }
+      }
```

Comment:
So after researching it a bit, I guess it might be a bit more complicated.
Looking at the table here: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollLeft#Browser_compatibility
it seems we have several options:

In RTL, the `scrollLeft` acts like this in different browsers:

| browser | scrollLeft direction |
| --- | --- |
| chrome | 0 <---> 100 |
| Edge(> v78) | 0 <--> 100 |
| Edge(<= v78) | 100 <--> 0 |
| ie | 100 <--> 0 |
| all the rest | -100 <--> 0 |

We'll need to find a solution that supports all cases
I'll try to think of something